content/blog/7-ways-to-deal-with-application-secrets-in-azure/index.md

```diff
@@ -0,0 +1,289 @@
+---
+title: "7 Ways to Deal with Application Secrets in Azure"
+authors: ["mikhail-shilkov"]
+tags: ["Azure", "Security"]
+date: "2019-07-26"
+
+meta_image: "feature.jpg"
+---
+
+Every non-trivial application relies on configuration values that may depend on the current execution environment. Some of these values contain sensitive information which shouldn't be shared publicly. The fewer parties have access to those secret values, the safer the application is. Ideally, no one should be granted access at all.
```

Comment:
Every non-trivial application relies on configuration values that may depend on the current execution environment. Some of these values contain sensitive information that shouldn't be shared publicly. In general, the fewer parties that have access to those secret values, the safer the application will be -- in fact, in an ideal world, no one would be granted direct access to those secrets.